saleor/graphql/account/mutations/permission_group.py

```diff
@@ -88,13 +89,16 @@ def clean_input(
     def clean_permissions(
         cls,
         requestor: "User",
+        group: auth_models.Group,
         errors: Dict[Optional[str], List[ValidationError]],
         cleaned_input: dict,
     ):
         field = "add_permissions"
         permission_items = cleaned_input.get(field)
         if permission_items:
             cleaned_input[field] = get_permissions(permission_items)
+            if requestor.is_superuser:
```

Comment:
Maby `if not requestor.is_superuser" ?

Reply:
Why? With opposite condition, it'll be more nesting.

Reply:
To not use empty 'return' :D

Reply:
Agree, this would look a bit better.